src/main/resources/frontend/src/views/StudentView/ResultsPreview.vue

```diff
@@ -34,9 +35,9 @@ const openDetails = ref<boolean>(false);
     </div>
     <p class="submission-notes" v-html="submission.notes.replace('\n', '<br />')"></p>
     <div class="rubric-item-summaries">
-      <p v-if="submission.rubric.passoffTests">Functionality: {{ roundTwoDecimals(submission.rubric.passoffTests.results.score) }} / {{ submission.rubric.passoffTests.results.possiblePoints }}</p>
-      <p v-if="submission.rubric.quality">Code Quality: {{ roundTwoDecimals(submission.rubric.quality.results.score) }} / {{ submission.rubric.quality.results.possiblePoints }}</p>
-      <p v-if="submission.rubric.unitTests">Unit Tests: {{ roundTwoDecimals(submission.rubric.unitTests.results.score) }} / {{ submission.rubric.unitTests.results.possiblePoints }}</p>
+      <p v-if="getPassoffTests(submission.rubric)">Functionality: {{ roundTwoDecimals(getPassoffTests(submission.rubric).results.score) }} / {{ getPassoffTests(submission.rubric).results.possiblePoints }}</p>
```

Comment:
Can we replace this with a dynamic data structure as well? It seems the only additional piece we'd need to store would be the human readable label ("Functionality", "Code Quality", "Unit Tests", etc...), and then we could streamline these declarations with a kind of iterator instead!